src/sections/node/channelsOutgoing.tsx

```diff
@@ -179,6 +180,66 @@ function ChannelsPage() {
       });
   };
 
+  const header = [
+    {
+      key: 'key',
+      name: 'key',
+    },
+    {
+      key: 'peerId',
+      name: 'Peer Id',
+      search: true,
+    },
+    {
+      key: 'status',
+      name: 'Status',
+      search: true,
+    },
+    {
+      key: 'funds',
+      name: 'Dedicated Funds',
+    },
+    {
+      key: 'actions',
+      name: 'Actions',
+      search: false,
+      width: '168px',
+      maxWidth: '168px',
+    },
+  ];
+
+  const parsedTableData = Object.entries(channels?.incoming ?? []).map(([, channel], key) => {
```

Comment:
We are filtering on the incoming, but it should be outgoing.
```suggestion
  const parsedTableData = Object.entries(channels?.outgoing ?? []).map(([, channel], key) => {
```

Reply:
good catch